### ISSUE TITLE
[Fluent UI -Suggested actions]: Keyboard focus is not visible on 'Previous and Next' icons while navigating with tab key.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5725]
### What is the PWD impact?

Keyboard‑only users may not know which control currently has focus. This makes it difficult to navigate the interface, interact with controls confidently, and complete actions efficiently. Users with low vision or motor disabilities are especially impacted.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Keyboard navigation

### How to reproduce the issue?

Repro Steps
1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to Fluent UI experience chat page and type Suggested-actions word in type your message text box.
4. Response will appear.
5. Navigate to the Next and Previous control.
6. Observe an issue that Keyboard focus is not visible on 'Previous and Next' icons while navigating with tab key.

### What do you expect?

The element should provide a clear, visible focus indicator when navigated to using the keyboard. 

### What actually happened?

Keyboard focus is not visible on 'Previous and Next' icons while navigating with tab key.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/0d9b6381-396f-4a58-8c10-ecb013358a87

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/focus-visible

### Adaptive Card JSON

```json

```
